admin/utils.php

```diff
@@ -245,7 +393,7 @@ public static function get_matching_rule( $object ) {
 						);
 					} else if ( $rule['rule_type'] === 'shared' ) {
 						return array(
-							'object_type' => 'transient',
+							'object_type' => 'option',
```

Comment:
Just curious--what is the reasoning behind using an option vs a transient for shared templates?

Reply:
The `get_transient` function will return `false` if the transient is expired. Since we want to avoid generation requests when a page is being loaded, and want those requests being triggered by a server side Cron, or else a WP Cron, I thought it best to store it as an option with a corresponding expiration, instead of as a transient.